### ISSUE TITLE
Different number format in different languages crash

### ISSUE THREAD

**Issue by saefbadwan** [ome450901/SimpleRatingBar#28]
```
 E/AndroidRuntime: FATAL EXCEPTION: main
                                                                           Process: com.network.imyth.network, PID: 25185
                                                                           java.lang.NumberFormatException: Invalid float: "٠.١٤"
                                                                               at java.lang.StringToReal.invalidReal(StringToReal.java:63)
                                                                               at java.lang.StringToReal.parseFloat(StringToReal.java:310)
                                                                               at java.lang.Float.parseFloat(Float.java:300)
                                                                               at com.willy.ratingbar.BaseRatingBar.calculateRating(BaseRatingBar.java:325)
                                                                               at com.willy.ratingbar.BaseRatingBar.handleMoveEvent(BaseRatingBar.java:316)
                                                                               at com.willy.ratingbar.BaseRatingBar.onTouchEvent(BaseRatingBar.java:292)
                                                                               at android.view.View.dispatchTouchEvent(View.java:7722)
                                                                               at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2234)
                                                                               at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1945)
                                                                               at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2240)
                                                                               at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1977)
                                                                               at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2240)
                                                                               at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1977)
                                                                               at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2240)
                                                                               at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1977)
                                                                               at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2240)
                                                                               at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1977)
                                                                               at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2240)
                                                                               at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1977)
                                                                               at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2240)
                                                                               at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1977)
                                                                               at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2240)
                                                                               at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1977)
                                                                               at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2240)
                                                                               at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1977)
                                                                               at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2240)
                                                                               at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1977)
                                                                               at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2240)
                                                                               at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1977)
                                                                               at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2240)
                                                                               at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1977)
                                                                               at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2240)
                                                                               at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1977)
                                                                               at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2240)
                                                                               at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:1977)
                                                                               at com.android.internal.policy.impl.PhoneWindow$DecorView.superDispatchTouchEvent(PhoneWindow.java:2108)
                                                                               at com.android.internal.policy.impl.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1542)
                                                                               at android.app.Activity.dispatchTouchEvent(Activity.java:2488)
                                                                               at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:68)
                                                                               at com.android.internal.policy.impl.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2056)
                                                                               at android.view.View.dispatchPointerEvent(View.java:7902)
                                                                               at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4011)
                                                                               at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:3890)
                                                                               at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3416)
                                                                               at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3466)
                                                                               at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3435)
                                                                               at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:3542)
                                                                               at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3443)
                                                                               at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:3599)
                                                                               at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3416)
                                                                               at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3466)
                                                                               at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3435)
                                                                           	at android.view.ViewRootImpl$InputStage.apply(ViewRootImp
```

**Comment by saefbadwan** [comment]
java.lang.NumberFormatException: Invalid float: "٠.١٤" that's equal 0.14  ,, the lib is getting number in the locals format, the error happens when clicking on a star

**Comment by ome450901** [comment]
Hello @saefbadwan  , 
Sorry reply so late, I haven't seen this Number format orz,
I'll do some research and repair this issue ASAP.
Thanks for your help!

**Comment by ome450901** [comment]
Fix in #56
